src/connector/src/parser/postgres.rs

```diff
@@ -163,48 +164,67 @@ pub fn postgres_row_to_owned_row(row: tokio_postgres::Row, schema: &Schema) -> O
                     }
                 }
                 DataType::Varchar => {
-                    match *row.columns()[i].type_() {
-                        // Since we don't support UUID natively, adapt it to a VARCHAR column
-                        Type::UUID => {
-                            let res = row.try_get::<_, Option<uuid::Uuid>>(i);
-                            match res {
-                                Ok(val) => val.map(|v| ScalarImpl::from(v.to_string())),
-                                Err(err) => {
-                                    if let Ok(suppressed_count) = LOG_SUPPERSSER.check() {
-                                        tracing::error!(
-                                            suppressed_count,
-                                            column = name,
-                                            error = %err.as_report(),
-                                            "parse uuid column failed",
-                                        );
-                                    }
-                                    None
+                    if let Kind::Enum(_) = row.columns()[i].type_().kind() {
```

Comment:
What is the `type_` when `kind` is `Kind::Enum`?

Reply:
`_ =>`, There isn't a type of `Type::ENUM`. And I cannot find a doc describing what `_ =>` includes.

Reply:
I mean the actual value of `type_`

Reply:
```
Inner::Other(Arc<Other>)
```